data/announcements/scheduled.yaml

```diff
@@ -174,3 +174,13 @@ announcements:
     k8s.gcr.io image registry is gradually being redirected to registry.k8s.io (since Monday March 20th).<br>
     All images available in k8s.gcr.io are available at registry.k8s.io.</br>
     Please read our [announcement](/blog/2023/03/10/image-registry-redirect/) for more details.
+
+- name: Freezing the legacy package repositories
+  startTime: 2023-09-01T00:00:00
+  endTime: 2023-11-04T00:00:00 # Until KubeCon NA 2023
+  style: "background: #c70202"
+  title: The legacy package repositories will be frozen starting with September 13, 2023
```

Comment:
```suggestion
  title: Changes to Linux package downloads for Kubernetes
```
I'd use a blue background. People don't actually need to act urgently on this, much as we might think it's a good idea.

Reply:
I tend to agree with the importance of making the change. From an accessibility standpoint, should there be some other thing to mark it as important for anyone that has colorblindness?

Reply:
What did we do w/ the registry.k8s.io  update communication?

Reply:
> However, I am wary of using the color too frequently. Overuse could get people weary of it, and this isn't a “fix your cluster now” announcement.

I tend to agree with this, we had the red announcement quite a few times this year, but I generally hope that we will not have to use the red announcement in the future that much or at all. While this change doesn't require action _right away_, it's **a very important change**. There are some additional aspects to this:

- It might be easy to miss this change because the legacy repositories will continue working for the foreseeable future, it's just that we'll stop publishing new versions to these repositories
- Users might just think that their version went EOL or something like that, rather than that the legacy repositories are deprecated
- We don't want to make users think that "this is something that can be easily changed in the future when needed". It's very important to migrate as soon as possible. Not being able to upgrade to a newer version can make you vulnerable very quickly, and in some cases, it might not be easy to migrate _that_ quickly

Reply:
> and this isn't a “fix your cluster now” announcement.

it's not an imminent, your cluster is broken sort of thing, but there will very likely be a non-trivial number of people that run into difficulties adapting to this, especially if they wait until it's time to do their next upgrade and they are skipping patch versions and land on one that will no longer be available. What about a yellow/orange background?

Reply:
Here's some experiment with orange background (`#d95e00`):

![image](https://github.com/kubernetes/website/assets/18719127/5224e322-d146-4607-9996-a831280cc84d)

Reply:
@jeremyrickard @sftim I pushed this change so that we can see how it looks like in preview.

Reply:
I like the idea of an orange banner because:
- red has been used too many times and the action is not a do it right now kind
- blue doesn't convey the idea of long term urgency